aptos-move/framework/aptos-framework/sources/transaction_fee.sepc.move

```diff
@@ -0,0 +1,15 @@
+spec aptos_framework::transaction_fee {
+    spec module {
+        pragma verify = true;
```

Comment:
why is `aborts_if_is_strict` not set? can you add the aborts_if spec for `burn_fee` here?

Reply:
Added `aborts_if_is_strict` and added spec to `burn_fee`.